src/Marquee/Marquee.js

```diff
@@ -950,7 +950,7 @@ var MarqueeItem = {
 		var node = this.$.marqueeText ? this.$.marqueeText.hasNode() : this.hasNode(),
 			rect;
 
-		if (node && this._marquee_distance == null && this.getAbsoluteShowing()) {
+		if (node && this.getAbsoluteShowing()) {
```

Comment:
Won't this mean that the marquee always re-measures? Seems like a major performance benefit loss. May be good to look into a way to invalidate `_marquee_distance` before we get here in those special cases where it's needed, so we keep the optimization.